gh-pr-create: add page (#6159)
tldr-pages/tldr: pages/common/gh-pr-create.md

# gh pr create

> Manage GitHub pull requests from the command-line.

> > More information: <https://cli.github.com/manual/gh_pr>.

Reviewer: ```suggestion
> More information: <https://cli.github.com/manual/gh_pr_create>.
```

> - Create a pull request:

Reviewer 2: ```suggestion
- Interactively create a pull request:
```

`gh pr create`

> - Create a pull request with out interactive shell:

Reviewer 2: ```suggestion
- Create a pull request, determining the title and description from the commit messages of the current branch:
```

> `gh pr create -f`

Reviewer: We prefer long arguments when are available:
```suggestion
`gh pr create --fill`
```

- Create a draft pull request:

`gh pr create -d`

> - Create a pull request:
> 
> `gh pr create -B base_branch -t 'Pull Request Title' -b 'Pull Request Description'`

Reviewer 2: ```suggestion
- Create a pull request, specifying the base branch, title, and description:

`gh pr create -B {{base_branch}} -t "{{pull_request_title}}" -b "{{pull_request_description}}"`
```

> - Open preview a pull request in the browser

Reviewer 2: ```suggestion
- Start opening a pull request in the browser:
```

> - Create a pull request, specifying the base branch, title, and description:

Reviewer 3: How about ?
```suggestion
- Create a pull request specifying the base branch, title, and description:
```

> `gh pr create -B {{base_branch}} -t "{{pull_request_title}}" -b "{{pull_request_description}}"`

Reviewer: ```suggestion
`gh pr create --base {{base_branch}} --title "{{title}}" --body "{{body}}"`
```

> - Start opening a pull request in the browser:

Reviewer 3: How about ?
```suggestion
- Create a pull request and open it in the browser:
```

Reviewer 2: Because it doesn't actually do that — it basically opens https://github.com/tldr-pages/tldr/compare only with your branch, so that you can configure everything in the UI.

Reviewer 4: `Open a pull request`

Reviewer 5: What's the problem with the original description? It reads better than any other suggested alteration here?
